bin/run_docker_dev.sh

```diff
@@ -53,7 +53,7 @@ if [[ $command == "build" ]]; then
 sudo chown -R 10001:10001 config/dev workspace repos
 sudo chown -R 10002:10002 workspace/logs/rabbitmq"
 elif [[ $command == "test" ]]; then
-    exec docker run -it \
+    exec docker run \
```

Comment:
Did you verify that this doesn't affect running locally?

Reply:
The suite runs fine for me locally, as they do not require any input. Taking the tty away from the "run" is an issue, but the tests just dump things into STDOUT and never use STDIN.